source/database-connections/manage-database-connections.txt

```diff
@@ -0,0 +1,86 @@
+.. _rm-manage-connections:
+
+===========================
+Manage Database Connections
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can copy, edit, or delete database connections from the Relational Migrator
+:guilabel:`Connections` page. Saved connections are listed by their environment
+tag. You can mouse over the :icon-fa5:`info-circle` icon for a connection to
+see a list of projects that use that connection.
+
+Copy a Connection
+-----------------
+
+You can copy an existing connection. This can be useful if you want multiple
+connections to the same server that each connect to different databases, or if
+you have similar connection information for a staging database and a 
+production database.
+
+Copying a connection creates a duplicate with ``-Copy`` appended to the
+connection name.
+
+
+Steps
+~~~~~
+
+To copy a connection:
+
+#. Click the :icon-fa4:`copy` button.
+
+#. Make any desired changes.
+
+#. Click :guilabel:`Save`.
+
+
+
+Edit a Connection
+-----------------
+
+You can edit an existing connection to change any settings except for the
+database type. This can be useful for:
+
+- Renaming the connection
+- Adding changing, or removing the :guilabel:`Environment tag`
+- Changing authentication credentials or SSL settings
+
+
+Steps
+~~~~~
+
+To edit a connection:
+
+#. Click the :icon-lg:`Edit` button.
+
+#. Make desired changes.
+
+#. Click :guilabel:`Save`.
+
+
+
+Delete a Connection
+-------------------
+
+You can delete a connection that you no longer need. Deleting a relational
+database connection causes any projects using that connection to fall back to
+Relational Migrator's last copy of the database contents. Mapping rules remain
+in place.
+
+Steps
+~~~~~
+
+To delete a connection:
+
+#. Click the :icon-fa5:`trash` button.
+
+   A warning dialog appears and lists any projects that use the connection.
```

Comment:
(suggestion)
This is a personal preference. I prefer "displays" over "appears" because appears makes the event sound a little magical.